packages/appstore/src/modules/traders-hub/index.tsx

```diff
@@ -1,19 +1,22 @@
 import React from 'react';
+import { observer } from 'mobx-react-lite';
 import CFDsListing from 'Components/cfds-listing';
 import ModalManager from 'Components/modals/modal-manager';
 import MainTitleBar from 'Components/main-title-bar';
+import TourGuide from 'Modules/tour-guide/tour-guide';
 import OptionsAndMultipliersListing from 'Components/options-multipliers-listing';
-import './traders-hub.scss';
-import { DesktopWrapper, MobileWrapper, ButtonToggle, Div100vhContainer } from '@deriv/components';
+import ButtonToggleLoader from 'Components/pre-loader/button-toggle-loader';
 import { useStores } from 'Stores/index';
-import { observer } from 'mobx-react-lite';
 import { isDesktop, routes } from '@deriv/shared';
-import ButtonToggleLoader from 'Components/pre-loader/button-toggle-loader';
+import { DesktopWrapper, MobileWrapper, ButtonToggle, Div100vhContainer, Button } from '@deriv/components';
+
+import './traders-hub.scss';
 
 const TradersHub = () => {
     const { traders_hub, client } = useStores();
     const { is_eu, is_landing_company_loaded } = client;
-    const { selected_platform_type, setTogglePlatformType } = traders_hub;
+    const { selected_platform_type, setTogglePlatformType, is_tour_open, toggleIsTourOpen, setIsOnboardingVisited } =
+        traders_hub;
```

Comment:
we do not need these `is_tour_open, toggleIsTourOpen, setIsOnboardingVisited` here anymore, please remove them